cmd/govim/vimstate.go

```diff
@@ -89,6 +89,17 @@ type vimstate struct {
 	// lastProgressText points to the text in the most recently created progress
 	// popup.
 	lastProgressText *strings.Builder
+
+	// vimgrepPendingBufs contain buffers read during a vimgrep quickfix command,
+	// keyed by buffer number.
+	// The purpose is to avoid sending DidOpen/DidClose notifications to gopls
+	// for all files that doesn't match the pattern.
+	// Current vim (8.2.2465) do reuse the buffer number for all non-matching
```

Comment:
"Current" has a fairly short-lived meaning. Rephrase perhaps?

Reply:
Hm, it was mostly to keep some kind of reference to a known version, i.e. "All versions, at least up until ... (might change in the future)". I dropped the version reference completely. It can still be tracked back if needed based on commit dates.